app/models/target.rb

```diff
@@ -10,7 +10,7 @@ class Target < ApplicationRecord
   validates :area_length, numericality: { greater_than: 0 }
   validate :user_must_have_less_than_maximum_targets
 
-  after_create :broadcast_to_compatible_users
+  after_save :broadcast_to_compatible_users
```

Comment:
Do you want to broadcast to all compatible users everytime you change something in a target?

Reply:
Ahhmmmm good catch. No 😢 
I just want to broadcast after an update to the new compatible targets.
But that would be a bit harder, wouldn't it?